api/src/services/users/users.test.ts

```diff
@@ -174,60 +184,179 @@ describe('user queries', () => {
 })
 
 describe('user writes', () => {
-  scenario('creates a user', async (scenario: StandardScenario) => {
-    mockCurrentUser({
-      id: scenario.user.one.id,
-      email: scenario.user.one.email,
-      roles: ['USDR_ADMIN'],
-    })
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
 
-    const result = await createUser({
-      input: {
-        email: 'uniqueemail2@test.com',
-        name: 'String',
-        agencyId: scenario.agency.one.id,
-        role: 'USDR_ADMIN',
-      },
+  describe('<Local Auth>', () => {
+    beforeAll(() => {
+      process.env.AUTH_PROVIDER = 'local'
     })
 
-    expect(result.email).toEqual('uniqueemail2@test.com')
-  })
+    scenario('creates a user', async (scenario: StandardScenario) => {
+      mockCurrentUser({
+        id: scenario.user.one.id,
+        email: scenario.user.one.email,
+        roles: ['USDR_ADMIN'],
+      })
+
+      const result = await createUser({
+        input: {
+          email: 'uniqueemail2@test.com',
+          name: 'String',
+          agencyId: scenario.agency.one.id,
+          role: 'USDR_ADMIN',
+        },
+      })
 
-  scenario('updates a user', async (scenario: StandardScenario) => {
-    mockCurrentUser({
-      id: scenario.user.one.id,
-      email: scenario.user.one.email,
-      roles: ['USDR_ADMIN'],
+      expect(result.email).toEqual('uniqueemail2@test.com')
     })
-    const email = 'String2@gmail.com'
-    const name = 'FDR'
-    const role = 'ORGANIZATION_STAFF'
-    const isActive = false
-    const original = (await user({ id: scenario.user.one.id })) as User
-    const result = await updateUser({
-      id: original.id,
-      input: { email, name, role, agencyId: scenario.agency.one.id, isActive },
+
+    scenario('updates a user', async (scenario: StandardScenario) => {
+      mockCurrentUser({
+        id: scenario.user.one.id,
+        email: scenario.user.one.email,
+        roles: ['USDR_ADMIN'],
+      })
+      const email = 'String2@gmail.com'
+      const name = 'FDR'
+      const role = 'ORGANIZATION_STAFF'
+      const isActive = false
+      const original = (await user({ id: scenario.user.one.id })) as User
+      const result = await updateUser({
+        id: original.id,
+        input: {
+          email,
+          name,
+          role,
+          agencyId: scenario.agency.one.id,
+          isActive,
+        },
+      })
```

Comment:
Removes use of `original` constant as it should be the same as `scenario.user.one`
```suggestion
      const result = await updateUser({
        id: scenario.user.one.id,
        input: {
          email,
          name,
          role,
          agencyId: scenario.agency.one.id,
          isActive,
        },
      })
```